cmd/provenanced/cmd/pre_upgrade_test.go

```diff
@@ -98,6 +98,9 @@ type cmdResult struct {
 }
 
 func executeRootCmd(t *testing.T, home string, cmdArgs ...string) *cmdResult {
+	// Ensure the keyring backend doesn't get changed when we run this command.
+	t.Setenv("PIO_TESTNET", "false")
+
```

Comment:
The default for this is `false`. So this line just makes sure its at its default value which it might not be if these tests are being run in my local environment that I also use to run various provenance commands, but has `PIO_TESTNET=true` in it.